asreview/analysis/analysis.py

```diff
@@ -0,0 +1,307 @@
+
+        if result_format == "percentage":
+            x_norm /= 100
+            y_norm /= 100
+        elif result_format == "number":
+            x_norm /= len(labels)
+            y_norm /= self.inc_found[fl]["inc_after_init"]
+
+        norm_xr = (np.arange(len(self.inc_found[fl]["avg"]))-dx)/x_norm
+        norm_yr = (np.array(self.inc_found[fl]["avg"])-dy)/y_norm
+        norm_y_err = np.array(self.inc_found[fl]["err"])/y_norm
+
+        return norm_xr, norm_yr, norm_y_err
+
+    def _get_inc_found(self, labels=False):
+        """Get the number of inclusions (without formatting)."""
+        inclusions_found = []
+
+        for logger in self.loggers.values():
+            inclusions, inc_after_init, n_initial = _find_inclusions(
+                logger, labels)
+            inclusions_found.append(inclusions)
+
+        inc_found_avg = []
+        inc_found_err = []
+        for i_instance in itertools.count():
+            cur_vals = []
+            for i_file in range(self.num_runs):
+                try:
+                    cur_vals.append(inclusions_found[i_file][i_instance])
+                except IndexError:
+                    pass
+            if len(cur_vals) == 0:
+                break
+            if len(cur_vals) == 1:
+                err = cur_vals[0]
+            else:
+                err = stats.sem(cur_vals)
+            avg = np.mean(cur_vals)
+            inc_found_avg.append(avg)
+            inc_found_err.append(err)
+
+        if self.num_runs == 1:
+            inc_found_err = np.zeros(len(inc_found_err))
+
+        return inc_found_avg, inc_found_err, inc_after_init, n_initial
+
+    def WSS(self, val=100, x_format="percentage", **kwargs):
+        """Get the WSS (Work Saved Sampled) value.
+
+        Arguments
+        ---------
+        val:
+            At which recall, between 0 and 100.
+        x_format:
+            Format for position of WSS value in graph.
+
+        Returns
+        -------
+        tuple:
+            Tuple consisting of WSS value, x_positions, y_positions of WSS bar.
+        """
+        norm_xr, norm_yr, _ = self.inclusions_found(
+            result_format="percentage", **kwargs)
+
+        if x_format == "number":
+            x_return, y_result, _ = self.inclusions_found(
+                result_format="number", **kwargs)
+            y_max = self.inc_found[False]["inc_after_init"]
+            y_coef = y_max/(len(self.labels) -
+                            self.inc_found[False]["n_initial"])
+        else:
+            x_return = norm_xr
+            y_result = norm_yr
+            y_max = 1.0
+            y_coef = 1.0
+
+        for i in range(len(norm_yr)):
+            if norm_yr[i] >= val - 1e-6:
+                return (norm_yr[i] - norm_xr[i],
+                        (x_return[i], x_return[i]),
+                        (x_return[i]*y_coef, y_result[i]))
+        return (None, None, None)
+
+    def RRF(self, val=10, x_format="percentage", **kwargs):
```

Comment:
Lowercase. Can you make those names more explicit?

Reply:
I updated the docstring of the rrf, I think using the abbreviations is okay, since scikit-learn does it in a similar way?